docs/_sass/minimal-mistakes/_page.scss

```diff
@@ -72,6 +72,23 @@
     border-bottom: 1px solid $border-color;
   }
 
+  h1, h2, h3, h4, h5, h6 {
+		.header-link {
```

Comment:
Just noticing that these lines use tabs instead of spaces as the surrounding parts of the document.

Reply:
🤦‍♂️😜